dockertest/environment.py

```diff
@@ -49,10 +46,10 @@ def get_selinux_context(path):
     :return: SELinux context as a string or None if not set
     """
     try:
-        # may include null-termination in string
-        return getxattr(path, 'security.selinux').replace('\000', '')
+        # First list item is null-terminated string length
+        return selinux.getfilecon(path)[1]
     except IOError, xcpt:
-        if xcpt.errno == errno.EOPNOTSUPP:
+        if xcpt.errno == errno.ENOENT:
```

Comment:
What's this change for? IIUC the EOPNOTSUPP was something xattr-specific. Now, using the selinux module, does this exception make sense? ISTM that feeding this function a bad path (ENOENT) should not be handled cleanly, it should be raised.